lib/puppet/provider/package/yum.rb

```diff
@@ -251,6 +251,7 @@ def latest
       return "#{upd[:epoch]}:#{upd[:version]}-#{upd[:release]}"
     else
       # Yum didn't find updates, pretend the current version is the latest
+      self.debug "Yum didn't find updates, current version(#{properties[:ensure]}) is the latest"
```

Comment:
nit: add a whitespace before the open paranthesis
```suggestion
      self.debug "Yum didn't find updates, current version (#{properties[:ensure]}) is the latest"
```